### DISCUSSION TITLE
add download progress visualization

### DISCUSSION THREAD

**Discussion by mutschler** [LeoKlaus/plappa#29] [Feature Requests]
maybe like a circular progressbar that fills up... so we can at least guess/see how long it would take to download the file to the device

**Comment by LeoKlaus** [comment]
There's already a progress indicator for downloads:
![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-06 at 10 10 58](https://github.com/LeoKlaus/plappa/assets/66492595/130eb548-7485-4aec-8b87-522257e2adca)
If you tap on the icon, you can see a list of all downloads with detailed progress information:
![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-06 at 10 11 14](https://github.com/LeoKlaus/plappa/assets/66492595/d7fa51e3-5d18-497c-b018-6f62726fda7f)

If you meant something else, please reopen!

**Comment by mutschler** [comment]
uhm, where's the first screenshot from? I didn't manage to get a view like that at least not that i could remember.

i however found the download icon in the top left of the status bar. which when clicked shows the progress list, but doesn't have an indicator that anything is still downloading. The icon stays the same no matter what. 

what i was proposing was a more visible indication on the List view. Something like this:
![image](https://github.com/LeoKlaus/plappa/assets/18465333/02141dd6-f8be-477c-b5f3-72751f2f7021)

But i guess just having the download icon (in the status bar) indicating a download going on would be good as well.
